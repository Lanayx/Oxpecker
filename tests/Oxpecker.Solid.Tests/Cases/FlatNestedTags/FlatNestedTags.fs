module Oxpecker.Solid.Tests.Cases.FlatNestedTags

open Oxpecker.Solid

[<SolidComponent>]
let Test () =
    body() {
        "zero"
        h1(id="one")
        div(id="two") { "2" }
        div() { "3" }
        br()
        "five"
    }
