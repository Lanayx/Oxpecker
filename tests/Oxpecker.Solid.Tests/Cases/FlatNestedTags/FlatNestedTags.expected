
export function Test() {
    return <body>
        zero
        <h1 id="one"></h1>
        <div id="two">
            2
        </div>
        <div>
            3
        </div>
        <br></br>
        five
    </body>;
}

